_data/navigation.yml

```diff
@@ -1,9 +1,12 @@
-- name: Use Solid
-  link: /use-solid
-- name: For Developers
-  link: /for-developers
-- name: For Enterprises
-  link: /for-enterprises
-- name: FAQ
-  link: /faqs
-
+- name: Developers
+  link: /developers
+- name: Users
+  link: /users
+- name: Newsletter
+  link: /newsletter
+- name: Events
+  link: /events
+- name: Press
+  link: /press
```

Comment:
I think we're going to have to cull some of these to fit on mobile screens (and to avoid stuffing too much in there in the first place). I'd argue that Press is the main one that doesn't meet the importance threshold, and possibly specification, just like web developers refer to MDN first over the HTML specs.

Reply:
We need some dropdown navigation items here. Didn't have time to get into that last night, but that's where it needs to go. FWIW - we hide the top navigation on mobile right now, but i'm assuming you mean once we have a burger menu there. As soon as we add dropdown menus these extras would go right under there. Was trying to avoid a sparse nav on desktop in the meantime.

Reply:
Oh, I intended to remove the comment about mobile, but apparently forgot - yes, I think it's mostly important to have some clear prioritisation here. We do hide them on mobile; all the links are available elsewhere as well. Generally, I'm of the opinion that drop-down menu and other kitchen-sink style menu's are usually a cop-out to satisfy different stakeholders who each want their pet favourite to be mentioned somewhere "important", rather focusing on what is actually important to the reader. Sparse navigation is a feature: it gives you a quick overview of the relevant sections that are likely to be of interest to you.

Reply:
There's probably some achievable follow-your-nose navigation that can avoid the dropdowns. It would certainly make the top-nav implementation simpler in jekyll 👍

Reply:
Removed press - lets keep drilling in on the follow-your-nose navigation in subsequent updates. I think you're right about avoiding the dropdowns.